ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/logging.j2

```diff
@@ -28,6 +28,12 @@ no logging monitor
 {%     elif logging.monitor is arista.avd.defined %}
 logging monitor {{ logging.monitor }}
 {%     endif %}
+{%     if logging.event.storm_control.discards.enable is arista.avd.defined(true) %}
+logging event storm-control discards global
```

Comment:
I wonder if we should change the key from `enable` to `global: <bool>`. Especially if it is independent from the `interval` (see other comment).
If not, it should probably be `enabled` instead to follow the same style as other keys.

Reply:
I think we should rename the `enable` key to `global` here.

Reply:
@ClausHolbechArista back from vacation and changes done